repo/packages/C/confluent-replicator/200/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "4.0",
+  "name": "confluent-replicator",
+  "version": "2.7.0-5.3.0",
+  "upgradesFrom": [
+    "2.6.0-5.1.2"
+  ],
+  "downgradesTo": [
+    "2.6.0-5.1.2"
+  ],
+  "scm": "https://github.com/confluentinc/cp-docker-images",
+  "description": "Confluent Enterprise Replicator for multi-data center synchronization of topic data\n\n\tDocumentation: https://docs.confluent.io/5.1.2/connect/connect-replicator/docs/index.html",
```

Comment:
same comment wrt to 5.3.0 docs

Reply:
good catch Sam! I updated them in rest of PRs also